### DISCUSSION TITLE
Welcome to Java - How to Program? Discussions Hub!

### DISCUSSION THREAD

**Discussion by hammadsaedi** [hammadsaedi/Java-How-to-Program#1] [Show and tell]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by farehiqbal** [comment]
Sir, suggest some project for intermediate Java developer. Happy to include Java swing in it too.
